pkg/commands/options/build.go

```diff
@@ -91,15 +92,15 @@ func (bo *BuildOptions) LoadConfig() error {
 	v.AddConfigPath(bo.WorkingDirectory)
 
 	if err := v.ReadInConfig(); err != nil {
-		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
-			return fmt.Errorf("error reading config file: %v", err)
+		if !errors.As(err, &viper.ConfigFileNotFoundError{}) {
```

Comment:
Can this be `Is`?

Reply:
Maybe. I couldn't figure out the correct invocation to get it to pass both lint and tests.

Reply:
Ok, then the only think I had was the CodeQL bit, which I think is a problem at HEAD.

Reply:
Yeah I'll tackle that one separately, I don't think it's a genuine issue at least, but convincing codeql might take some work.